src/app/modules/document-detail/document-create/document-attachment/document-attachment.component.ts

```diff
@@ -50,10 +55,12 @@ export class DocumentAttachmentComponent implements OnInit {
   constructor(
     private readonly translateService: TranslateService,
     private readonly supportedMimeTypeV1Service: SupportedMimeTypeControllerV1APIService,
-    private readonly dataSharingService: DataSharingService
+    private readonly dataSharingService: DataSharingService,
+    private userService: UserService
   ) {}
 
   ngOnInit(): void {
+    this.translateService.use(getBestMatchLanguage(this.userService));
```

Comment:
Not needed if you use addInitializeModuleGuard. It is not safe in all cases to do it here!